infrasim/model.py

```diff
@@ -2146,8 +2145,9 @@ def init(self):
             racadm_obj.set_log_path("/var/log/infrasim/{}/racadm.log".
                                     format(self.__node_name))
 
-            if "racadm_port" in self.__node:
-                racadm_obj.set_port_idrac(self.__node["racadm_port"])
+            if "racadm" in self.__node:
+                if "port" in self.__node["racadm"]:
+                    racadm_obj.set_port_idrac(self.__node["racadm"]["port"])
```

Comment:
@JuneZhou19 line 2148-2150 is doing the same thing with line 2020.
I prefer to keep 2020, and remove 2148-2050 together with function `set_port_idrac`:

- 2020 is more infrasim model init style to set default value
- while 2148-2150 leverage a function `set_port_idrac`, which is not necessarily needed. You can see serveral `set_xxx` in model, but this kind of "set_xxx" is typical to set attribute to a instance while its definition must be outside of this instance's scope, one example is: `bmc_connection_port` is defined in top level of yml file, it shall be synced to both CBMC and CCompute, so there would be `set_port_qemu_ipmi` in CBMC and CCompute